planning/planning_evaluator/launch/planning_evaluator.launch.xml

```diff
@@ -2,11 +2,21 @@
   <arg name="input/trajectory" default="/planning/scenario_planning/trajectory"/>
   <arg name="input/reference_trajectory" default="/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/trajectory"/>
   <arg name="input/objects" default="/perception/object_recognition/objects"/>
+  <arg name="launch_diagnostic_converter" default="false"/>
 
-  <node name="planning_evaluator" exec="planning_evaluator" pkg="planning_evaluator" output="screen">
-    <param from="$(find-pkg-share planning_evaluator)/param/planning_evaluator.defaults.yaml"/>
-    <remap from="~/input/trajectory" to="$(var input/trajectory)"/>
-    <remap from="~/input/reference_trajectory" to="$(var input/reference_trajectory)"/>
-    <remap from="~/input/objects" to="$(var input/objects)"/>
-  </node>
+  <!-- planning evaluator -->
+  <group>
+    <node name="planning_evaluator" exec="planning_evaluator" pkg="planning_evaluator">
+      <param from="$(find-pkg-share planning_evaluator)/param/planning_evaluator.defaults.yaml"/>
+      <remap from="~/input/trajectory" to="$(var input/trajectory)"/>
+      <remap from="~/input/reference_trajectory" to="$(var input/reference_trajectory)"/>
+      <remap from="~/input/objects" to="$(var input/objects)"/>
+      <remap from="~/metrics" to="/diagnostic/planning_evaluator/metrics"/>
+    </node>
+  </group>
+
+  <!-- diagnostic converter -->
```

Comment:
converter node should be called from the planning.launch, not in the planning_evaluator.launch